game/src/index.ts

```diff
@@ -69,14 +80,58 @@ const write: PlayerMove<GS, WritePayload> = {
     }
 
     board.answers = {};
-
+    board.playerVotes = {};
+    const allAnswers: string[] = [];
     for (const [userId, playerboard] of Object.entries(playerboards)) {
+      let checkedAnswers: string[] = [];
       const playerAnswers = playerboard.answers[board.round];
-      board.answers[userId] = playerAnswers;
+      board.answers[userId] = playerAnswers.map((a) => {
+        if (
+          playerAnswers.filter(
+            (answer) => answer.toLowerCase() === a.toLowerCase(),
+          ).length > 1
+        ) {
+          const valid = checkedAnswers.includes(a) ? false : true;
+          checkedAnswers.push(a);
+          return {
+            answer: a,
+            valid: valid,
+          };
+        } else {
+          return {
+            answer: a,
+            valid: true,
+          };
+        }
+      });
+
+      allAnswers.push(...playerAnswers);
+    }
+    allAnswers.forEach((a) => {
+      board.playerVotes[a] = {};
+      for (const [userId] of Object.entries(playerboards)) {
+        board.playerVotes[a][userId] = 1;
+      }
+    });
```

Comment:
See the new proposed `votes` datastructure!

```suggestion
// Initialize the votes to 1 points each.
// TODO: deal with 2-pointers
const votesForOneAnswer = Object.fromEntries(userIds.map((userId) => [userId, 1]))
board.votes = board.categories.map(() => Object.fromEntries(
  userIds.map((userId) => [userId, {...votesForOneAnswer}])
));
```

Not sure if there is a simpler way 🤔